src/TEC/Plugin.php

```diff
@@ -0,0 +1,392 @@
+<?php
+/**
+ * Plugin Class.
+ *
+ * @since   1.0.0
+ *
+ * @package TEC\Extensions\WpaiAddOn
+ */
+
+namespace TEC\Extensions\WpaiAddOn;
+
+use TEC\Common\Contracts\Service_Provider;
+use TEC\Extensions\WpaiAddOn\Import\Post_Handler;
+
+/**
+ * Class Plugin
+ *
+ * @since   1.0.0
+ *
+ * @package TEC\Extensions\WpaiAddOn
+ */
+class Plugin extends Service_Provider {
+	/**
+	 * Stores the version for the plugin.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @var string
+	 */
+	const VERSION = '1.0.0';
+
+	/**
+	 * Stores the base slug for the plugin.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @var string
+	 */
+	const SLUG = TEC_EXTENSION_WPAI_ADD_ON_SLUG;
```

Comment:
Do we use this anywhere?

Reply:
Also, what's the benefit of storing a constant in a constant? Shorter string?

Reply:
Oh... no a side-effect of a refactor without thinking about it. I'll fix it. Good point.